Added link to the playlist (#54)
CodingTrain/Intro-to-Data-APIs-JS: README.md

# Working with Data and APIs in JavaScript

> This is a working document outlining future plans for a new Coding Train playlist: ["Working with Data and APIs in JavaScript"](https://www.youtube.com/watch?v=DbcLg8nRWEg&list=PLRqwX-V7Uu6YxDKpFzf_2D84p0cyk4T7X&index=1)

Reviewer: ```suggestion
This document is an outline for the Coding Train playlist: ["Working with Data and APIs in JavaScript"](https://www.youtube.com/watch?v=DbcLg8nRWEg&list=PLRqwX-V7Uu6YxDKpFzf_2D84p0cyk4T7X&index=1)
```
